Resources/Prototypes/_Nuclear14/Entities/Structures/Misc/radio.yml

```diff
@@ -18,4 +18,140 @@
         mask:
         - MachineMask
         layer:
-        - LowImpassable
\ No newline at end of file
+        - LowImpassable
+
+- type: entity #prototype for radio station microphone
+  parent: BaseStructure
+  id: RadioDJMicrophone
+  name: radio microphone
+  description: An decent quality professional microphone for use in DJ booths.
+  components:
+  - type: Sprite
+    sprite: _Nuclear14/Structures/Misc/radiostation/microphones.rsi
+    state: radiomicrophone
+  - type: RadioMicrophone
+    broadcastChannel: PBS
+    toggleOnInteract: false # Nuclear-14
+    frequency: 1667 # Nuclear-14
+    listenRange: 2
+  - type: Clickable
+  - type: InteractionOutline
+  - type: Appearance
+  - type: WiresVisuals
+  - type: Transform
+    noRot: false
+    anchored: true
+  - type: ActiveRadio
+    channels:
+    - PBS
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeCircle
+            radius: 0.20
+        density: 1000
+        mask:
+        - MachineMask
+        layer:
+        - MachineLayer
+  # Nuclear-14-Start
+  - type: ActivatableUI
+    key: enum.HandheldRadioUiKey.Key
+  - type: UserInterface
+    interfaces:
+      - key: enum.HandheldRadioUiKey.Key
+        type: HandheldRadioBoundUserInterface
+  # Nuclear-14-End
```

Comment:
This whole section is for N14, so you don't need the `# Nuclear-14` comments

Reply:
those were artifacts from copy/pasting the code, I'll remove and push